homework/week1/RosmaryFC/HoroscopeHW20150508/app/src/main/java/nyc/c4q/rosmaryfc/horoscopehw20150508/MainActivity.java

```diff
@@ -0,0 +1,127 @@
+package nyc.c4q.rosmaryfc.horoscopehw20150508;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import java.util.Locale;
+
+
+public class MainActivity extends Activity{
+private String[] mSignTitles;
+private DrawerLayout mDrawerLayout;
+private ListView mDrawerList;
+
+private ActionBarDrawerToggle mDrawerToggle;
+private CharSequence mDrawerTitle;
+private CharSequence mTitle;
+
+        @Override
+        protected void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            setContentView(R.layout.activity_main);
+
+            mSignTitles = getResources().getStringArray(R.array.signs_array);
+            mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+            mDrawerList = (ListView) findViewById(R.id.left_drawer);
+
+            // Set the adapter for the list view
```

Comment:
Once again, I like that there are useful comments that explain the code that follows.